app/src/main/java/com/sevtinge/cemiuiler/module/hook/guardprovider/DisableUploadAppList.java

```diff
@@ -26,7 +26,7 @@ public void init(/*final XC_LoadPackage.LoadPackageParam lpparam*/) {
             if (guardApplication != null) {
                 Field[] guardApplicationFields = guardApplication.getDeclaredFields();
                 for (Field field : guardApplicationFields) {
-                    if (field.getName().equals("c")) {
+                    if ("c".equals(field.getName())) {
```

Comment:
interesting that the reordering of this prevent possible null pointer exceptions, nice